packages/elements-core/src/components/Docs/HttpService/ServerInfo.tsx

```diff
@@ -25,25 +27,25 @@ export const ServerInfo: React.FC<ServerInfoProps> = ({ servers, mockUrl }) => {
     <InvertTheme>
       <Panel rounded isCollapsible={false} className="BaseURLContent" w="full">
         <Panel.Titlebar whitespace="nowrap">API Base URL</Panel.Titlebar>
-        <Box overflowX="auto">
-          <Panel.Content w="full" className="sl-flex sl-flex-col">
-            <VStack spacing={1} divider>
-              {serversToDisplay.map((server, index) => (
-                <ServerUrl {...server} key={index} />
-              ))}
-            </VStack>
-          </Panel.Content>
-        </Box>
+        <Panel.Content w="full" className="sl-flex sl-flex-col">
+          <VStack spacing={1} divider>
+            {serversToDisplay.map((server, index) => (
+              <ServerUrl {...server} key={index} />
+            ))}
+          </VStack>
+        </Panel.Content>
       </Panel>
     </InvertTheme>
   );
 };
 
-const ServerUrl = ({ description, url, marginBottom = true }: IServer & { marginBottom?: boolean }) => {
+const ServerUrl = ({ id, description, url }: IServer & { marginBottom?: boolean }) => {
```

Comment:
not sure if intended from the UI standpoint or not, but you lost the default value for `marginBottom`.
It was `marginBottom = true` in the prior version

Reply:
Hmm i actually don't even see where it was used?

Reply:
let's remove it altogether than haha